README.md

```diff
@@ -1,33 +1,103 @@
-# Getting Started with Create React App
-
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
-
-In the project directory, you can run:
-
-### `npm start`
-
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
-
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [🛠 Built With](#built-with)
+  - [Tech Stack](#tech-stack)
+- [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+  - [🤝 Contributing](#contributing)
+  - [⭐️ Show your support](#support)
+  - [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# Space-travelers-hub
+This application displays the Covid19 Updates Statistical Data for all Continents in the World and you can also search by Continent name.
+
+# 🛠 Built With <a name="built-with"></a>
+ > This project was built with; 
+- HTML 
+- CSS
+- Javascript
+- React
+- Redux
+- Jest
+
+# Tech Stack <a name="tech-stack"></a>
+> ### Technologies used 
+- GitHub 
+- Visual Studio Code 
+- GitBash
+- Webpack
+<details> 
+<summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+# Key Features <a name="key-features"></a>
+- **Makes Space Mission Reservation**
+- **Displays available Rockets, Missions and Dragons**
+- **Cancels /Leaves reservations and joined missions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# Live-Demo
+> Here is a link (https://blase147.github.io/react-captone-hub/) to the live page.
```

Comment:
- [ ] The demo link you provided does not really work. Kindly deploy your app and provide the correct link.

  ![fufufp](https://user-images.githubusercontent.com/81468675/206129774-a41f0d8e-430e-4e31-b103-77120f666265.jpg)

- [ ] I cannot find the link to your video presentation on the readme file. Kindly record a video presenting your project and remember to highlight the piece of code you find interesting.